json_spec/README.md

```diff
@@ -1,16 +1,2589 @@
+
+For example a [LESS_THAN](#validatortype-lessthan) validator will validate that a [_measurement_](#measurement) is "less than" the comparison value.
+
+<table>
+    <tr>
+        <td><strong>Value</strong></td>
+        <td><strong>Description</strong></td>
+    </tr>
+    <tr>
+        <td><strong>EQUAL</strong></td>
+        <td>Validates that a <a href="#measurement"><em>measurement</em></a> and a comparison value are equal.</td>
+    </tr>
+    <tr>
+        <td><strong>NOT_EQUAL</strong></td>
+        <td>Validates that a <a href="#measurement"><em>measurement</em></a> and a comparison value are not equal.</td>
+    </tr>
+    <tr>
+        <td><a name="validatortype-lessthan"><strong>LESS_THAN</strong></a></td>
+        <td>Validates that a <a href="#measurement"><em>measurement</em></a> is less than a comparison value.</td>
+    </tr>
+    <tr>
+        <td><strong>LESS_THAN_OR_EQUAL</strong></td>
+        <td>Validates that a <a href="#measurement"><em>measurement</em></a> is less than or equal to a comparison value.</td>
+    </tr>
+    <tr>
+        <td><strong>GREATER_THAN</strong></td>
+        <td>Validates that a <a href="#measurement"><em>measurement</em></a> is greater than a comparison value.</td>
+    </tr>
+    <tr>
+        <td><strong>GREATER_THAN_OR_EQUAL</strong></td>
+        <td>Validates that <a href="#measurement"><em>measurement</em></a> is greater than or equal to a comparison value.</td>
+    </tr>
+    <tr>
+        <td><strong>REGEX_MATCH</strong></td>
+        <td>Validates that a <a href="#measurement"><em>measurement</em></a> is found to match one or more regex strings contained by the <a href="#validator"><em>validator</em></a>.</td>
+    </tr>
+    <tr>
+        <td><strong>REGEX_NO_MATCH</strong></td>
+        <td>Validates that a <a href="#measurement"><em>measurement</em></a> does not match any regex strings contained by the <a href="#validator"><em>validator</em></a>.</td>
+    </tr>
+    <tr>
+        <td><strong>IN_SET</strong></td>
+        <td>Validates that a <a href="#measurement"><em>measurement</em></a> exists within a set of allowable values contained by the <a href="#validator"><em>validator</em></a>.</td>
+    </tr>
+    <tr>
+        <td><strong>NOT_IN_SET</strong></td>
+        <td>Validates that a <a href="#measurement"><em>measurement</em></a> does not exist within a set of allowables values contained by the <a href="#validator"><em>validator</em></a>.</td>
+    </tr>
+</table>
+
+
+### Identifiers
+
+Due to the streaming nature of the [**specification**](#def-spec), a common implementation pattern is the adoption of an identifier to indicate a relationship between an emitted artifact and a feature of the [**system under test**](#def-dut). Examples include identifiers for hardware components and software elements of the [**device under test**](#def-dut).
```

Comment:
This reads a bit odd since we have both system and device under test which point to the same definition.  Should we just stick with 'device under test' everywhere?

Reply:
done. please check/resolve